roles/cluster-inventory/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Get floating IP of nodes in a Magnum cluster using Openstack CLI
+  shell: "openstack coe cluster show {{ cluster_name }} -c master_addresses -c node_addresses -f json"
+  register: result_addresses
+
+- debug:
+    var: ansible_user
+
+- name: Get a list of all running servers using Openstack CLI
+  shell: "openstack server list -c ID -c Networks -f json --user {{ ansible_user }}"
```

Comment:
Should this be a variable the user can set? What If I want to run this playbook against your cluster?

Reply:
Leave my clusters alone.